File: urbanairship-preference-center-react-native/android/src/main/java/com/urbanairship/reactnative/preferenceCenter/AirshipPreferenceCenterPackage.java
package com.urbanairship.reactnative.preferenceCenter;

import java.util.Arrays;
import java.util.Collections;
import java.util.List;

import com.facebook.react.ReactPackage;
import com.facebook.react.bridge.NativeModule;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.uimanager.ViewManager;

public class AirshipPreferenceCenterPackage implements ReactPackage {

    @Override
    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
        return Arrays.<NativeModule>asList(new AirshipPreferenceCenterModule(reactContext));
    }

    @Override
    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
        return Collections.emptyList();
    }
}

File: urbanairship-preference-center-react-native/android/src/main/java/com/urbanairship/reactnative/preferenceCenter/AirshipPreferenceCenterPackage.kt
/* Copyright Urban Airship and Contributors */

package com.urbanairship.reactnative.preferenceCenter

import com.facebook.react.ReactPackage
import com.facebook.react.bridge.ReactApplicationContext
import com.facebook.react.bridge.NativeModule
import com.facebook.react.uimanager.ViewManager

class AirshipPreferenceCenterPackage : ReactPackage {

    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
        return listOf<NativeModule>(AirshipPreferenceCenterModule(reactContext))
    }

    override fun createViewManagers(reactContext: ReactApplicationContext): List<ViewManager<*, *>> {
        return emptyList()
    }
}